version-history.md

```diff
@@ -9,6 +9,7 @@ permalink: /version-history/
 
 OpenSearch version | Release highlights | Release date
 :--- | :--- | :--- | :---
+[2.0.0](https://github.com/opensearch-project/opensearch-build/blob/main/release-notes/opensearch-release-notes-2.0.0.md) | Adds document-level monitors for alerting, OpenSearch Notifications plugins, and Geo Map Tiles in OpenSearch Dashboards. Also adds support for Lucene 9 and bug fixes for all OpenSearch plugins. | 24 May 2022
```

Comment:
Should we say, "Includes..." instead of "Adds..."? Primarily because we have a lot more changes than what is listed. Or end with "For a full list of release highlights, see the Release Notes."?